### PR TITLE
sui: tilt updates

### PR DESCRIPTION
**Pull request by jumpsiegel**

### CODE REVIEW

File: sui/Docker.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # first build the image
-(cd ..; DOCKER_BUILDKIT=1 docker build --progress plain -f sui/Dockerfile.base -t sui .)
+#(cd ..; DOCKER_BUILDKIT=1 docker build --no-cache --progress plain -f sui/Dockerfile.base -t sui .)
```
Comment (hendrikhofstadt) [inline sui/Docker.md:2]: why not use the cache?

**Review by hendrikhofstadt — COMMENTED** [review]
What's the rationale behind the port change?

File: Tiltfile [hunk 2]
```diff
@@ -395,7 +395,6 @@ if solana:
         port_forwards = [
             port_forward(8899, name = "Solana RPC [:8899]", host = webHost),
             port_forward(8900, name = "Solana WS [:8900]", host = webHost),
-            port_forward(9000, name = "Solana PubSub [:9000]", host = webHost),
```
[the remark this answers is not in the document]
Comment (jumpsiegel) [reply Tiltfile:398]: None of the tilt tests use the websocket stuff..

File: sui/Docker.md (hunk 1 above)
Comment (jumpsiegel) [reply sui/Docker.md:2]: When they do a new release,  I flush the cache while building a docker base image so really this line is used for cut/paste

File: Tiltfile (hunk 2 above)
[the remark this answers is not in the document]
Comment (kcsongor) [reply Tiltfile:420]: can I resolve this? not super familiar with this part of the setup

**Pull request merged**